public/i18n/de.json

```
@@ -70,6 +70,10 @@
   "form.download.download-programmatically-info.url": "https://opendatadocs.meteoswiss.ch/de/general/download",
   "form.download.download-programmatically-info": "Informationen, wie Sie Daten automatisch beziehen können",
   "form.collection-selection.title": "Messnetz wählen",
+  "form.collection-selection.snow-hint.first-text": "Für die",
+  "form.collection-selection.snow-hint.url": "https://opendatadocs.meteoswiss.ch/de/general/faq#what-snow-measurement-data-is-available-and-where-can-i-find-it",
```

Der deutsche Link hat einen anderen Anchor
```suggestion
  "form.collection-selection.snow-hint.url": "https://opendatadocs.meteoswiss.ch/de/general/faq#schneemessdaten--welche-daten-gibt-es-und-wo-finde-ich-sie",
```

---

Ist das nur in der deutschen Sprache so?

---

Ja, bei den anderen fehlt der Text noch, und Andreas muss dann den Anchor übernehmen. Ist also i.O. so